.evergreen/config.yml

```diff
@@ -172,11 +172,29 @@ functions:
           SWIFT_VERSION=${SWIFT_VERSION} \
             sh ${PROJECT_DIRECTORY}/.evergreen/install-dependencies.sh
 
+  "upload-mo-artifacts":
+    - command: shell.exec
+      params:
+        script: |
+          ${PREPARE_SHELL}
+          find $MONGO_ORCHESTRATION_HOME -name \*.log | xargs tar czf mongodb-logs.tar.gz
+    - command: s3.put
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: mongodb-logs.tar.gz
+        remote_file: ${UPLOAD_BUCKET}/${build_variant}/${revision}/${version_id}/${build_id}/logs/${task_id}-${execution}-mongodb-logs.tar.gz
+        bucket: mciuploads
+        permissions: public-read
+        content_type: ${content_type|application/x-gzip}
+        display_name: "mongodb-logs.tar.gz"
+
 pre:
   - func: "fetch source"
   - func: "install dependencies"
 
 post:
+  - func: "upload-mo-artifacts"
```

Comment:
post is run after *every* task, which may present problems for our tasks that do not actually use mongo-orchestration (e.g. the Atlas tests may fail because this directory won't exist).
the duplication is annoying but it might be better to add this manually to each task which calls "stop mongo-orchestration"

Reply:
done